chainerx_cc/chainerx/float16.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <cstdint>
+
+namespace chainerx {
+
+class Half {
+public:
+    Half() {}
+    Half(const Half& v) : data_{v.data_} {}
+    explicit Half(float v);
```

Comment:
How about implementing these constructors inline?

Reply:
Constructors from float/double cannot be made inline, because the function bodies are implemented in the `.cc` file.